pyiron_base/generic/datacontainer.py

```diff
@@ -56,6 +57,7 @@ def _parse_yaml(file_name):
             warnings.warn(exc)
             return {}
 
+
 def _parse_xml(self, file_name, wrap=False):
```

Comment:
```suggestion
def _parse_xml(file_name, wrap=False):
```
Is there are particular reason why the parse functions are not static functions of the class.

Reply:
I didn't want to clutter the interface too much.  If you don't want the function outside the class, I can define them inside the `read()`.